index.html

```diff
@@ -0,0 +1,132 @@
+      </div>
+    </header>
+    <main>
+      <section class="top-bottom-section part-1">
+        <div>
+          <h2 class="sub-title">
+            일상의 모든 물건을 <br />
+            거래해 보세요
+          </h2>
+          <a href="/items.html">구경하러 가기</a>
+        </div>
+        <div>
+          <img src="imgs/Img_home_top.png" alt="언덕위 판다" />
+        </div>
+      </section>
+      <section class="main-section part-2">
+        <div>
+          <img src="imgs/Img_home_01.png" alt="판다 이미지 1" />
+        </div>
+        <div>
+          <div class="highlight">Hot itme</div>
+          <h2 class="sub-title">
+            인기 상품을 <br />
+            확인해 보세요
+          </h2>
+          <p class="description">
+            가장 HOT한 중고거래 물품을 <br />
+            판다 마켓에서 확인해 보세요
+          </p>
+        </div>
+      </section>
+      <section class="main-section part-3">
+        <div>
+          <div class="highlight">Search</div>
+          <h2 class="sub-title">
+            구매를 원하는<br />
+            상품을 검색하세요
+          </h2>
+          <p class="description">
+            구매하고 싶은 물품은 검색해서 <br />
+            쉽게 찾아보세요
+          </p>
+        </div>
+        <div>
+          <img src="imgs/Img_home_02.png" alt="판다 이미지 2" />
+        </div>
+      </section>
+      <section class="main-section part-4">
+        <div>
+          <img src="imgs/Img_home_03.png" alt="판다 이미지 3" />
+        </div>
+        <div>
+          <div class="highlight">Register</div>
+          <h2 class="sub-title">
+            판매를 원하는 <br />
+            상품을 등록하세요
+          </h2>
+          <p class="description">
+            어떤 물건이든 판매하고 싶은 상품을<br />
+            쉽게 등록하세요
+          </p>
+        </div>
+      </section>
+      <section class="top-bottom-section part-5">
+        <div>
+          <h2 class="sub-title">
+            일상의 모든 물건을 <br />
+            거래해 보세요
+          </h2>
+        </div>
+        <div>
+          <img src="imgs/Img_home_bottom.png" alt="언덕위 판다" />
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="footer-container">
+        <small class="footer-copy">© codeit · 2024</small>
+        <nav class="footer-nav" aria-label="푸터 내비게이션">
+          <ul class="footer-links" role="list">
+            <li><a href="/privacy.html">Privacy Policy</a></li>
+            <li><a href="/faq.html">FAQ</a></li>
+          </ul>
+        </nav>
+        <ul class="footer-links" role="list">
+          <li>
+            <a href="https://www.facebook.com/?locale=ko_KR"
+              ><img src="imgs/ic_facebook.png"
+            /></a>
+          </li>
+          <li>
+            <a href="https://x.com/"><img src="imgs/ic_twitter.png" /></a>
+          </li>
+          <li>
+            <a href="https://www.youtube.com/"
+              ><img src="imgs/ic_youtube.png"
+            /></a>
+          </li>
+          <li>
+            <a href="https://www.instagram.com/"
+              ><img src="imgs/ic_instagram.png"
+            /></a>
```

Comment:
💊 제안
현재 코드에서는 태그와 태그가 들여쓰기 없이 한 줄로 붙어 있어 가독성이 조금 떨어질 수 있습니다.  
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~

```suggestion
            <a href="https://www.instagram.com/">
              <img src="imgs/ic_instagram.png"/>
            </a>
```